Library/Homebrew/rubocops/conflicts.rb

```diff
@@ -15,11 +15,25 @@ class Conflicts < FormulaCop
           bash-completion@2
         ].freeze
 
-        def audit_formula(_node, _class_node, _parent_class_node, body)
+        def audit_formula(_node, _class_node, _parent_class_node, body_node)
+          conflicts_with_node = find_node_method_by_name(body_node, :conflicts_with)
+          return unless conflicts_with_node
+
+          if parameters(conflicts_with_node).last.respond_to? :values
+            reason = string_content(parameters(conflicts_with_node).last.values.first)
+            first_word = reason.split.first
+
+            if reason.match?(/\A[A-Z]/)
+              problem "'#{first_word}' from the conflicts_with reason should be '#{first_word.downcase}'."
```

Comment:
Very minor but you might want to surround `conflicts_with` with backticks.

This formats it differently then the normal text and, in my opinion, makes it easier to read.

```suggestion
              problem "'#{first_word}' from the `conflicts_with` reason should be '#{first_word.downcase}'."
```

Reply:
This was lifted directly from `keg_only.rb`, so someone might want to apply this change there also.

Reply:
Completely agree. Thanks for pointing that out!

Reply:
It looks like the changes I suggested here could also be added to `keg_only.rb`, so I'll work on a PR to implement those there.